config/application.rb

```diff
@@ -1,53 +1,39 @@
-# frozen_string_literal: true
-
-require_relative 'boot'
-
-require 'action_cable/engine'
-require 'action_controller/railtie'
-# require 'action_mailbox/engine'
-require 'action_mailer/railtie'
-# require 'action_text/engine'
-require 'action_view/railtie'
-require 'active_job/railtie'
-require 'active_record/railtie'
-require 'active_storage/engine'
-require 'rails/test_unit/railtie'
+require_relative "boot"
+
+require "rails"
```

Comment:
This looks new. I don't think we want to require all of rails given we're pulling in the bits we need below.